autometa/taxonomy/ncbi.py

```diff
@@ -149,6 +149,8 @@ def name(self, taxid, rank=None):
         try:
             taxid = self.convert_taxid_dtype(taxid)
         except DatabaseOutOfSyncError:
+            err_message = f"Databases out of sync. Taxid ({taxid}) not found in nodes.dmp or names.dmp!"
+            logger.error(err_message)
```

Comment:
The `err_message` is already being emitted by `DatabaseOutOfSyncError`. 

```suggestion
        except DatabaseOutOfSyncError as err:
            logger.error(err)
```